common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -1,199 +1,156 @@
-                            style={
-                              canvas.thumbnail.service.width <=
-                              canvas.thumbnail.service.height
-                                ? {
-                                    width: 'auto',
-                                    height: '300px',
-                                    marginRight: '12px',
-                                  }
-                                : {
-                                    width: '300px',
-                                    height: 'auto',
-                                    marginRight: '12px',
-                                  }
-                            }
-                            src={
-                              i < 4
-                                ? iiifImageTemplate(
-                                    canvas.thumbnail.service['@id']
-                                  )({
-                                    size: `!${previewSize},${previewSize}`,
-                                  })
-                                : null
-                            }
-                            data-src={
-                              i > 3
-                                ? iiifImageTemplate(
-                                    canvas.thumbnail.service['@id']
-                                  )({
-                                    size: `!${previewSize},${previewSize}`,
-                                  })
-                                : null
-                            }
-                          />
-                        </a>
-                      );
-                    })}
-                  </div>
-                </ScrollContainer>
-              )}
-              {/* TODO temporary links to large image for ttesting, while we don't have a viewer */}
-              <div>
-                <Button
-                  type="primary"
-                  url={iiifImageTemplate(
-                    sequence.canvases[0].thumbnail.service['@id']
-                  )({ size: '!1024,1024' })}
-                  text={`View all ${sequence.canvases.length} images`}
-                  icon="gallery"
-                />
-              </div>
-            </Fragment>
-          )}
-        </Fragment>
-      ))}
+      {imageThumbnails &&
+        imageThumbnails.map(pageType => {
+          return pageType.images.map(image => (
+            <img
+              style={{ width: 'auto', maxHeight: '300px' }}
+              key={image.uri}
+              src={image.uri}
+            />
+          ));
+        })}
     </Fragment>
   );
 };
 
 export default IIIFPresentationDisplay;
-// TODO image alt - how do we handle this
-// TODO if we go with single image then show specific page if available title page, then cover, then first image (preferably first text image, not blank page)
-// TODO import IIIFBookPreview?
+// TODO image alt - how do we handle this - no need images are just presentational now, so add correct aria
```

Comment:
Thought on structured images to not make the preview too heavy is to just say something like `Title page of Alice in wonderland`.